src/components/report-viewer/log-report.js

```diff
@@ -80,6 +82,14 @@ export default class LogReport {
         return this.logs;
     }
 
+    getSeverityList() {
+        return this.severityList;
+    }
+
+    getAllSeverityList() {
```

Comment:
```suggestion
    // Represent all the different severities of this report and it's subreports
    getAllSeverityList() {
```
I would add something like that

Reply:
I added it in the constructor and not on the getter